superduperdb/base/datalayer.py

```diff
@@ -88,15 +86,15 @@ def __init__(
         self.fast_vector_searchers = LoadDict(
             self, callable=self.initialize_vector_searcher
         )
-        self.distributed = s.CFG.mode == 'production'
+        self.distributed = s.CFG.mode == Mode.Production
```

Comment:
not needed anymore, the compute has the information implicitly 

compute.submit() will do it distributed compute if production mode else local

so not needed

Reply:
If I remove it, everything breaks due to missing distributed field.

Reply:
distributed is required to be removed from the project entirely 

@blythed